### DISCUSSION TITLE
Multiple Files For Code Editor

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2525] [Suggestions]
This is a killer feature that Boostnote has. The idea is that you create a Code Editor entry but you create multiple tabs in the same entry to add multiple files. When it comes to HTML bundles etc, Standardnotes is useless for the task at the moment, would be fantastic if this was coming.

**Comment by atmoio** [comment]
Would be neat!

**Comment by AYapejian** [comment]
This could go hand-in-hand with #469

**Comment by ghost** [comment]
@AYapejian Totally.
